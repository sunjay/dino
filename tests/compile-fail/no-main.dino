// This function is not main
fn foo() {}
