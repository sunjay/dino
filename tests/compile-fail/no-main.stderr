Error: In 'tests/compile-fail/no-main.dino': `main` function not found
